bin/install.sh

```diff
@@ -3,4 +3,5 @@
 (
 cd "$( dirname "${BASH_SOURCE[0]}" )/.."
 pip install -r requirements.txt
+pip install -r dev.requirements.txt
```

Comment:
In my experience pip will respect TXT changes for all packages that have versions specified, which IMO should be all of them